Gemfile

```diff
@@ -1,19 +1,20 @@
 source "http://rubygems.org"
 
-gem "aws-sdk-ec2", "~> 1"
```

Comment:
Hmmm weird that it wasn't removed in the Rails upgrade - could you take a look and double check that we didn't decide to leave it in for some strange reason. Or if you have just ignore me :)

Reply:
We didn't touch this repo during the first round as it's not using Rails